### DISCUSSION TITLE
Album management and multi tagging songs

### DISCUSSION THREAD

**Discussion by fasteddys** [Arlodotexe/strix-music#211] [Q&A] [answered]
### Describe the problem

1) For user Is there a way to group the collection of songs or book readings into an album

2) Can you allow us to see the [for e,g, any free or transcribe ](https://www.assemblyai.com/blog/easy-c-speech-recognition/)the text and view it along with song as its playing

In many cases people want to see the words as they learn to sing.

### Describe the proposed change

1) For user Is there a way to group the collection of songs or book readings into an album

2) Can you allow us to see the [for e,g, any free or transcribe ](https://www.assemblyai.com/blog/easy-c-speech-recognition/)the text and view it along with song as its playing

In many cases people want to see the words as they learn to sing.

**Comment by Arlodotexe** [comment, marked as answer]
@fasteddys I've converted this to a QnA discussion.

1. Albums are published releases created by artists. If the user wants to create a collection of songs (or audiobooks, we can support that) then they should create a playlist. We do support playlist folders as well.

2. We have support for Lyrics in the SDK. These can be supplied by the core, or by a plugin. You could create a plugin that transcribes the audio and converts it to timed lyrics, which could synchronize with any audio player.

**Comment by papyr** [comment]
If I could chime in as well, I also would like to see something to allow kareoke, for .g. for the lyrics, in some cases if people want to do a little bit of karaoke or something along those lines and let's say they had an international song.

In my case, i use a third-party service to transcribe it and I would like to display their lyrics along with the song player similar to a karaoke and highlight the songs as the player moves along the timeline.

hope that makes sense... Just chiming in

**Reply by Arlodotexe** [reply]
This is something we can accommodate. You can create a plugin for w track that adds lyrics. This can be with whatever lyrics provider you want.

Additionally, you can multiple of these with different lyrics providers, and apply them all. The first plugin in the load order will be invoked first, then the next one, etc., until it reaches the non-plugin version with data provided by a core.
